livekit-server/values.yaml

```diff
@@ -104,3 +104,5 @@ securityContext:
 tolerations: []
 
 affinity: {}
+
+strategy: {}
```

Comment:
nit: `strategy` is abstract at the root level, can we call this `deploymentStrategy` and include a comment linking to:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#deploymentstrategy-v1-apps

Reply:
@davidzhao thanks for the comment, I've corrected the PR.